src/Uno.UI.RuntimeTests.Engine.Library/Engine/UI/UnitTestsControl.cs

```diff
@@ -100,7 +100,7 @@ public UnitTestsControl()
 			GetContent: () => unitTestContentRoot.Content as UIElement,
 			SetContent: elt => unitTestContentRoot.Content = elt
 		);
-		UnitTestsUIContentHelper.CurrentTestWindow = XamlWindow.Current;
+		UnitTestsUIContentHelper.CurrentTestWindow ??= XamlWindow.Current;
```

Comment:
@dr1rrb This doesn't look right.

`CurrentTestWindow` getter will throw if the field is null. So, attempting to read it here will fail.